src/_shared/scss/_adverts-books.scss

```diff
@@ -73,6 +73,44 @@
         }
     }
 
+    &.advert--prominent-true > .advert__image-container {
+        flex-basis: calc(66.67% - #{$gs-gutter / 2});
+    }
+
+    &.has-no-flex {
```

Comment:
Thanks for the knowledge! As it turns out, this actually reasserts the same flex-basis that is on the flex container already so I can just remove it. It must have been left over from when I was playing around with the spacing.